test/extension/handlers/topologymutation/handler.go

```diff
@@ -250,6 +255,9 @@ func patchKubeadmConfigTemplate(_ context.Context, obj runtime.Object, _ map[str
 	// 1) Set extraArgs
 	switch obj := obj.(type) {
 	case *bootstrapv1beta1.KubeadmConfigTemplate:
+		if obj.Spec.Template.Spec.JoinConfiguration == nil {
+			obj.Spec.Template.Spec.JoinConfiguration = &bootstrapv1beta1.JoinConfiguration{}
+		}
 		if obj.Spec.Template.Spec.JoinConfiguration.NodeRegistration.KubeletExtraArgs == nil {
 			obj.Spec.Template.Spec.JoinConfiguration.NodeRegistration.KubeletExtraArgs = map[string]string{}
 		}
```

Comment:
Not here but line 258 is wrong?

It appends from InitConfiguration but should be JoinConfiguration?!